aptos-move/framework/aptos-framework/sources/stake.spec.move

```diff
@@ -602,10 +656,12 @@ spec aptos_framework::stake {
     spec add_stake_with_cap {
         include ResourceRequirement;
         let amount = coins.value;
+        aborts_if reconfiguration_state::spec_is_in_progress();
         include AddStakeWithCapAbortsIfAndEnsures { amount };
     }
 
     spec add_stake {
+        aborts_if reconfiguration_state::spec_is_in_progress();
```

Comment:
so we still don't have apply to all except in the spec?

Reply:
let's see what @rahxephon89  says

Reply:
Will revisit all related cases after it is merged to main